vue-components/src/components/bouncingDots.ts

```diff
@@ -0,0 +1 @@
+export const bouncingDotsSizes = [ 'small', 'medium' ];
```

Comment:
It might be that things get confused by having the same filename that differs only by extension and capitalization. Better to rename this file to `bouncingDotsProps.ts` or something like that

Reply:
yea makes sense to change the name